src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double sideBig;
+    private double sideShort;
+    private double height;
+
+    public IsoscelesTrapezoid(double sideBig, double sideShort, double height) {
+        super(new ColorSupplier().getRandomColor());
+        if (sideShort > 0 && sideBig > 0 && height > 0 && sideBig > sideShort) {
+            this.sideBig = sideBig;
+            this.sideShort = sideShort;
+            this.height = height;
+        }
+
```

Comment:
[Please don’t add redundant empty lines to your code.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#please-dont-add-redundant-empty-lines-to-your-code)

Reply:
Ok) andestand. I fix it